core/cron/pokedex.rarity.php

```diff
@@ -46,15 +42,8 @@
 	$pokelist[$key]['rate'] = $rounded;
 }
 
-// create new array if file doesn't exist
-// else use file content
-if (!is_file($pokedex_rarity_file)) {
-	$pokemons_rarity = new stdClass();
-} else {
-	$pokemons_rarity = json_decode($pokedex_rarity_file_content);
```

Comment:
The only reason is to keep pokes with 0 rarity but I am not sure if it is needed